platform/api/server/list/endpoint.go

```diff
@@ -194,6 +204,20 @@ func MakeGetDEPProfileEndpoint(svc Service) endpoint.Endpoint {
 	}
 }
 
+func MakeBackupEndpoint(svc Service) endpoint.Endpoint {
+	return func(ctx context.Context, request interface{}) (response interface{}, err error) {
+		data, err := svc.Backup(ctx)
+		return backupResponse{Data: data, Err: err}, nil
+	}
+}
+
+type backupRequest struct{}
+
+type backupResponse struct {
+	Data []byte `json:"data"`
```

Comment:
I have an idea for how to handle this as an io.Reader instead. let's handle that after the initial cleanup comments.